eth_bots/main.py

```diff
@@ -6,8 +6,8 @@
 
 from eth_typing import BlockNumber
 
-from examples.eth_bots.setup_experiment import setup_experiment
-from examples.eth_bots.trade_loop import trade_if_new_block
+from eth_bots.setup.setup_experiment import setup_experiment
+from eth_bots.setup.trade_loop import trade_if_new_block
```

Comment:
`trade_loop` doesn't seem like a `setup` thing. maybe `core` is a better descriptor?